### PR TITLE
Fix: minor spelling errors

### PR DESCRIPTION
**Pull request by joeashcraft**
Minor spelling errors, normalized on "American English" in accordance with [Ansible documentation style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#id3)

### CODE REVIEW

File: structures/README.adoc [hunk 1]
```diff
@@ -10,10 +10,10 @@ Explanations::
 define for which use case to use roles, playbooks, potentially workflows (in Ansible Controller/Tower/AWX), and how to split the code you write.
 
 Rationale::
-especially when writing automation in a team, it is important to have a certain level of consistence and make sure everybody has the same understanding.
+especially when writing automation in a team, it is important to have a certain level of consistency and make sure everybody has the same understanding.
 By lack of doing so, your automation becomes unreadable and difficult to grasp for new members or even for existing members.
 +
-This structure will also help you to have a consistent level of modelization so that re-usability becomes easier.
+This structure will also help you to have a consistent level of modularization so that re-usability becomes easier.
```
Comment (mophahr) [inline structures/README.adoc:16]: i think _modelization_ is actually correct here but I agree that it is an uncommon way of putting it. How about changing it to

```suggestion
Following a consitent structure will increse re-usability.
```

**Review by mophahr — COMMENTED** [review]
One minor suggestion. Other than that, LGTM :)

File: structures/README.adoc (hunk 1 above)
Comment (mophahr) [reply structures/README.adoc:16]: @joeashcraft could you please have another look at this so we can approve it next week.

Comment (joeashcraft) [reply structures/README.adoc:16]: @mophahr I like yours better! It much easier to read. I'll copy your wording and fix a small typo in an update to this PR.

**Review by ericzolf — APPROVED** [review]
Looks good to me. Thanks and sorry for the delay.

**Review by mophahr — APPROVED** [review]

**Comment by mophahr** [comment]
Approved in CoP meeting

**Review by sean-m-sullivan — APPROVED** [review]

**Pull request merged**